sdk/applicationinsights/Microsoft.Azure.ApplicationInsights/src/Microsoft.Azure.ApplicationInsights.Query.csproj

```diff
@@ -10,6 +10,10 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <RootNamespace>Microsoft.Azure.ApplicationInsights.Query</RootNamespace>
   </PropertyGroup>
+  <!-- Disable StyleCop -->
+  <PropertyGroup>
```

Comment:
Similar to the test projects I think I would opt-out all the track 1 libraries by default in the common props/targets.